### PR TITLE
Password policy doc 1335

### PR DESCRIPTION
**Pull request by lennessyy**

### CODE REVIEW

File: modules/security/pages/password-policy.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+To enable the password policy, run the following commands:
+
+[.wrap,console]
+----
+$ gadmin config set Security.UserPasswordPolicy.Enable true <1>
+$ gadmin config set Security.ACLPasswordPolicy.Enable true  <2>
+$ gadmin config apply
+$ gadmin restart gsql
+----
+<1> Setting `Security.UserPasswordPolicy.Enable` to true applies the password policy to passwords for TigerGraph database users.
```
Comment (QXG2) [inline modules/security/pages/password-policy.adoc:29]: also add them to https://docs.tigergraph.com/tigergraph-server/current/reference/configuration-parameters

And mention that the policy won't disable existing user with simple password.

Comment (lennessyy) [reply modules/security/pages/password-policy.adoc:29]: yes, they'll be added as I run a script to produce all the new config parameters for every version

Comment (QXG2) [reply modules/security/pages/password-policy.adoc:29]: Still need to mention that
* the policy won't disable existing user with simple password.
* mandatory password policy than always exist: password can't be empty string ("") or "null".

Comment (lennessyy) [reply modules/security/pages/password-policy.adoc:31]: Added to the draft

**Review by alexthomasTG — APPROVED** [review]

**Pull request merged**